src/sidebar.c

```diff
@@ -724,6 +725,16 @@ static void create_openfiles_popup_menu(void)
 			G_CALLBACK(on_openfiles_document_action), GINT_TO_POINTER(OPENFILES_ACTION_RELOAD));
 	doc_items.reload = item;
 
+	item = gtk_image_menu_item_new_with_mnemonic(_("R_ename or Save As..."));
```

Comment:
We use *Save As* alone in the rest of the UI, should we introduce *Rename* here?